cmd/kubectl-gadget/fsslower.go

```diff
@@ -41,7 +41,7 @@ var fsslowerCmd = &cobra.Command{
 	Short: "Trace open, read, write and fsync operations slower than a threshold",
 	PreRunE: func(cmd *cobra.Command, args []string) error {
 		if fsslowerFilesystem == "" {
-			return utils.WrapInErrMissingArgs("--filesystem")
+			return utils.WrapInErrMissingArgs("--type")
```

Comment:
```suggestion
			return utils.WrapInErrMissingArgs("--type / -t")
```

For consistency with `utils.WrapInErrInvalidArg` below, line 58? Or use `"--type"` everywhere.

Reply:
Makes sense, thanks!